gnmi/gnmi_test.go

```diff
@@ -71,9 +71,13 @@ func TestHandleMetrics(t *testing.T) {
 				},
 				metricCh: make(chan *pmetric.Metrics, 10),
 			}
-			var n *gpb.Notification
+
+			var cnt int
 			updateFn := func(notif *gpb.Notification) error {
-				n = notif
+				cnt++
```

Comment:
The `duplicates` here should only be increased if there is an update for the same path. I fixed the test logic error in #17 too in a way which I think preserves the contents in case we did want to assert something about them.

Reply:
nice! incorporated.